File: src/main/java/no/nav/fo/veilarbregistrering/domain/besvarelse/Stilling.java
package no.nav.fo.veilarbregistrering.domain.besvarelse;

import lombok.*;
import lombok.experimental.Accessors;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Accessors(chain = true)
@ToString
@EqualsAndHashCode
public class Stilling {
    private String label;
    private long konseptId;
    private String styrk08;
}

File: src/main/java/no/nav/fo/veilarbregistrering/besvarelse/Stilling.kt
package no.nav.fo.veilarbregistrering.besvarelse

data class Stilling(
    val label: String? = null,
    val konseptId: Long = 0,
    val styrk08: String? = null,
) {
    override fun toString(): String {
        return "Stilling(label=$label, konseptId=$konseptId, styrk08=$styrk08)"
    }
}

fun tomStilling(): Stilling {
    return Stilling("", -1L, "-1")
}

val ingenYrkesbakgrunn: Stilling = Stilling("X", -1L, "X")
